go.mod

```diff
@@ -29,4 +29,4 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 
 replace github.com/cosmos/ledger-cosmos-go => github.com/crypto-com/ledger-cosmos-go v0.9.10-0.20200929055312-01e1d341de0f
 
-replace github.com/cosmos/cosmos-sdk => github.com/crypto-com/cosmos-sdk v0.41.0-patched-tm
+replace github.com/cosmos/cosmos-sdk => ./third_party/cosmos-sdk
```

Comment:
it points to master branch?

Reply:
it's the same tag, the commit hash can be reviewed [here](https://github.com/crypto-org-chain/chain-main/pull/384/files#diff-000ed18c511ee5e37e33a1aeb8fa9a2ab9a9b149dcec5fc3e58f36f845ebc610)